command-processor/tests/FileCommandsProcessorTests.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+from src.controllers.implementations.FileCommandsProcessor import \
+FileCommandsProcessor
+
+
+class FileCommandsProcessorTest(TestCase):
+    def setUp(self):
+        self._user_id = 'test_user_id'
+        self._file_cp = FileCommandsProcessor()
+
+    def tearDown(self):
+        pass
+
+    def test_create(self):
+        _result = self._file_cp.create(self._user_id, 'some_file.xmind')
+        self.assertTrue(_result)
```

Comment:
is this enough or should we check that file was created or at least FileOperations.create() was called?

Reply:
thank you for reviewing but PR is not ready for review yet :)